index.js

```diff
@@ -1,24 +1,54 @@
 'use strict';
 
 const env = require('env-var');
-const assert = require('assert');
 const raygun = require('raygun');
+const log = require('tke-logger').getLogger(__filename);
 
+// If FH_ENV is one of these values, then RAYGUN_API_KEY must be present
+const REQUIRED_ENVS = [
```

Comment:
If the FH_ENV is one of these values, then we'll throw an error if there's no api key set